x-pack/plugin/esql/qa/testFixtures/src/main/resources/qstr-function.csv-spec

```diff
@@ -0,0 +1,48 @@
+###############################################
+# Tests for QSTR function
```

Comment:
I would recommend to add a few more complicated tests here, some combinations of conjunctive and disjunctive predicates, qrst with other functions - functions that can be pushed down into Lucene and also have compute engine implementation(like cider_match), functions that cannot be pushed down into Lucene and but have compute engine implementation(like length), perhaps pick/duplicate a few from the new tests added in LocalPhysicalPlanOptimizerTests and added here, as this is the place we validate correct results.

Reply:
Understood! I did that in a3f859fa718ca48d0f6cbdddb436db468918d4d8